### DISCUSSION TITLE
Multi-kernel support

### DISCUSSION THREAD

**Discussion by webbedfeet** [quarto-dev/quarto-cli#496] [Q&A] [answered]
Hello,

Currently in RMarkdown with reticulate, we can create documents with both R and Python chunks. My use case is teaching materials where we use both R and Python in showing how to do something. Is this currently possible in Quarto?

**Comment by jjallaire** [comment, marked as answer]
Yes! Use Knitr and reticulate just as you currently do with R Markdown (i.e. Quarto uses Knitr under the hood any document with R chunks -- in this case the Python chunks will automatically be handled by reticulate)
